### PR TITLE
[feature](expr) Add length limit to the debug string of expr.

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

Not merge to 2.0 currently, if we merge happenlee's PR

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

File: be/src/vec/exprs/vcast_expr.cpp [hunk 1]
```diff
@@ -142,8 +142,11 @@ std::string VCastExpr::debug_string() const {
             out << ",";
         }
         out << input_expr->debug_string();
```
Comment (yiguolei) [inline be/src/vec/exprs/vcast_expr.cpp:144]: 这么改感觉像是每个expr < 4096,但是如果是多个expr 还是会崩溃。
我认为，我们应该给debug_string 传递一个(stringstream) 然后判断啊

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Pull request closed without merging**